tox.ini

```diff
@@ -25,13 +29,17 @@ commands=
     pip install --require-hashes -r requirements.txt --target ./package
     pip install --no-deps --target ./package .
 
+    # Install packages from private repositories
+    pip install --target ./package git+https://{env:GITHUB_TOKEN:}@github.com/release-engineering/cdn-definitions-private.git#egg=cdn_definitions_private
```

Comment:
Can we please add --no-deps here as well?  The goal is to ensure there's no untrusted installs of indirect dependencies (e.g. if cdn-definitions-private declared a dependency on something which didn't appear in requirements.txt of this repo).